testdata/project-v2-addon/go.mod

```diff
@@ -9,5 +9,5 @@ require (
 	k8s.io/apimachinery v0.17.2
 	k8s.io/client-go v0.17.2
 	sigs.k8s.io/controller-runtime v0.5.0
-	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200317144824-bbf1fb2a4a9a
+	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200413154948-0284e6ed8028
```

Comment:
not sure why this changed, just did it because tests was failing

Reply:
It was not because of your PR. Could you please rebase it with the master?

Reply:
Also, could we not add a test for its scenario in https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/model/resource/options_test.go? WDYT?

Reply:
merged!



> Also, could we not add a test for its scenario in https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/model/resource/options_test.go? WDYT?

not sure I get it... you meant, add a test to `builds` pluralize thing?